src/python/bcc/table.py

```diff
@@ -412,9 +412,37 @@ def __init__(self, *args, **kwargs):
         raise Exception("Unsupported")
 
 class StackTrace(TableBase):
+    MAX_DEPTH = 127
```

Comment:
This value is hardcoded in the kernel. I am not sure whether it'd be worth it to dig into the FFI ctypes to figure it out dynamically. Thoughts?